segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/DebugBookKeeperLogWrapper.java

```diff
@@ -54,11 +56,13 @@
 public class DebugBookKeeperLogWrapper implements DebugDurableDataLogWrapper {
     //region Members
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(DebugBookKeeperLogWrapper.class);
```

Comment:
Please, use `@Slf4j` annotation in the class as in the rest of classes instead of instantiating the logger this way.

Reply:
added slf4j logger.